submissions/iakow/friends-app/js/main.js

```diff
@@ -0,0 +1,232 @@
+"use strict";
+
+disableFilters();
+
+const USERS_AMOUNT = 100;
+
+mountUserList(getPlaceholders());
+
+const ALL_USER_CARDS = [];
+
+fetch(`https://randomuser.me/api/?results=${USERS_AMOUNT}`)
+  .then((response) => response.json())
+  .then((response) => {
+    response.results.forEach((userData) => {
+      ALL_USER_CARDS.push(getUserCard(userData));
```

Comment:
Because the array is empty at this stage.

Reply:
I see that array is empty. But you don't answered on my second question. I think with map your code will be more clean.
You can do think like this:
```
let ALL_USER_CARDS = []
...
.then(({ results }) => {
  ALL_USER_CARDS = results.map(getUserCard);
}
```

Reply:
Yes, I already figured it out and implemented it in the local repository. I haven't made a push yet, because I want to reformat all the code related to receiving data via API, where promises are used, including error handling, in accordance with your comments. I suddenly realized that I needed to learn more about promises) Thanks